Expensify/App
# Add lastMessageText to a report to prep chat preview line issue

### Details
This is a quick preparation step for adding the chat preview line to the UI.

### Fixed Issues (Part 1) 
https://github.com/Expensify/Expensify/issues/152633

### Tests
1. Run the app
2. Inspect localStorage for `report_` keys
3. Verify that the keys have a `lastMessageText` field with the last message from a user

### Tested On


### Screenshots
❌
